packages/marshal/src/rankOrder.js

```diff
@@ -142,16 +142,20 @@ export const makeComparatorKit = (compareRemotables = (_x, _y) => 0) => {
       case 'string': {
         // Within each of these passStyles, the rank ordering agrees with
         // JavaScript's relational operators `<` and `>`.
+        // @ts-expect-error FIXME narrowed
```

Comment:
Just curious if you have a plan for how we can eventually get these dispatches to do the narrowing for us. Everything I thought of is worse that just leaving these `@ts-expect-error FIXME`s in place.

Reply:
There's something in TS for narrowing by `typeof` but I don't expect they'll our helper function for pass style. The pragmatic thing may be to drop to `any` in places where the code is doing its own type validation.

Reply:
Thx. Resolving.